example/src/example_layer.cpp

```
@@ -34,31 +39,156 @@ bool TestLayer::OnEvent(moth_ui::Event const& event) {
+        m_graphics.SetTarget(m_target.get());
+
+        m_graphics.SetBlendMode(canyon::graphics::BlendMode::Replace);
+        m_graphics.SetColor({ 0.05f, 0.05f, 0.15f, 1.0f });
+        m_graphics.Clear();
+
+        // Eight spinning coloured lines radiating from the centre
+        float const cx = 128.0f, cy = 128.0f, radius = 110.0f;
+        float const angle = m_elapsedMs * 0.001f;
+        m_graphics.SetBlendMode(canyon::graphics::BlendMode::Add);
+        for (int i = 0; i < 8; i++) {
+            float t = angle + i * (3.14159f / 4.0f);
```

Clang (14.0.6)

[error] narrowing conversion from 'int' to 'float'

(cppcoreguidelines-narrowing-conversions,-warnings-as-errors)